tests/dialects/test_bigquery.py

```diff
@@ -476,9 +483,8 @@ def test_bigquery(self):
             },
             write={
                 "bigquery": "SELECT * FROM UNNEST(['7', '14']) AS x",
-                "presto": "SELECT * FROM UNNEST(ARRAY['7', '14']) AS (x)",
-                "hive": "SELECT * FROM UNNEST(ARRAY('7', '14')) AS (x)",
-                "spark": "SELECT * FROM UNNEST(ARRAY('7', '14')) AS (x)",
+                "presto": "SELECT * FROM UNNEST(ARRAY['7', '14']) AS _t(x)",
+                "spark": "SELECT * FROM UNNEST(ARRAY('7', '14')) AS _t(x)",
```

Comment:
Two notes here:

- The presto test was previously invalid, i.e. `AS (x)` is invalid syntax:
```
trino> SELECT * FROM UNNEST(ARRAY['7', '14']) AS (x);
Query 20231024_194935_00002_hyjxp failed: line 1:43: mismatched input '('. Expecting: <identifier>
```
- The hive/spark tests were also invalid, for two reasons: the `(x)` is invalid syntax and there's no `UNNEST` function; we need to use `EXPLODE` but this needs some work and we'd better keep it out of this PR's scope.